### PR TITLE
Fixed changes mentioned in issues

### PR DESCRIPTION
**Pull request by sahenrai**

### CODE REVIEW

File: Bruin Bite/View Controllers/MenuVC.swift [hunk 1]
```diff
@@ -38,6 +38,17 @@ class MenuVC: UIViewController, UICollectionViewDelegate, UICollectionViewDataSo
     let diningHalls = ["De Neve", "BPlate", "Covel", "Feast"]
     var menuItem: Item?
     
+    var loading_1: UIImage!
+    var loading_2: UIImage!
+    var loading_3: UIImage!
+    var loading_4: UIImage!
+    var loading_5: UIImage!
+    var loading_6: UIImage!
+    var images: [UIImage]!
```
Comment (kamalca) [inline Bruin Bite/View Controllers/MenuVC.swift:47]: You shouldn't need to make six variables to build an array. Just use an array and a for loop to build the array of images.

File: Bruin Bite/View Controllers/MenuVC.swift [hunk 2]
```diff
@@ -46,13 +57,27 @@ class MenuVC: UIViewController, UICollectionViewDelegate, UICollectionViewDataSo
...
+        
+        imageAnim.image = animatedImage
+        
         self.view.backgroundColor = UIColor.white
         
         super.viewDidLoad()
         
         NetworkManager.isUnreachable { networkManagerInstance in
             
-           self.loadingLabel.text = "No internet!"
```
Comment (kamalca) [inline Bruin Bite/View Controllers/MenuVC.swift]: You took out this line but left in an empty function. It appears that this feature no longer requires the Reachability functionality, so why are we leaving it in our code base?

**Comment by kamalca** [comment]
Make sure that your commits always have useful names and clarifying descriptions.
Your code should always be concise, remember that the purpose of an array is so that you don't have to give a name to every variable.
When changing your implementation of a feature, be sure to remove any code that is no longer being used. We don't want to leave useless code in our project as it makes it hard to understand what is happening.

**Comment by HirdayGupta** [comment]
Closed  due to a long period of inactivity. Also the feature needed from this PR was implemented in #141

**Pull request closed without merging**